### PR TITLE
ioinformatics

### PR DESCRIPTION
**Pull request by Desipientia**

### CODE REVIEW

File: app/adapters/ioinformatics.js [hunk 1]
```diff
@@ -20,7 +20,11 @@ CATS.Adapter.IOInformatics = Classify({
         for (var i = 0; i < this.contest.problems.length; ++i) {
             var prob = result_table.get_empty_problem_for_score_board_row();
             prob.problem = this.contest.problems[i];
-            prob.points = $(tds[i + 3]).text();
+            prob.points = parseFloat($(tds[i + 3]).text());
```
Comment (klenin) [inline app/adapters/ioinformatics.js]: Why not `parseInt` ?

File: app/rules/school.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ CATS.Rule.School = Classify(CATS.Rule.Base, {
         var teams_arr = _.map(teams, function (v, team_id) {
             var points_cnt = 0;
             for (var i = 0; i < contest.problems.length; ++i)
-                points_cnt += teams_problems[team_id][i].points;
+                points_cnt += parseFloat(teams_problems[team_id][i].points);
```
Comment (klenin) [inline app/rules/school.js]: Why no NaN check? Or maybe `parseFloat` should be called earlier?

**Comment by klenin** [comment]
Merged anyway, it is mostly ok.

File: app/rules/school.js (hunk 2 above)
Comment (Desipientia) [inline app/rules/school.js]: It is called earlier in ioinformatics.js, line 23, with NaN check. Sorry, I haven't notice that.

**Pull request merged**